docs/api-reference/cloud-cli.md

```diff
@@ -198,6 +201,8 @@ It executes the [migration commands declared in dbos-config.yaml](./configuratio
 
 **Parameters:**
 - `[application-name]`: The name of the application to deploy. By default we obtain the application name from package.json. This argument overrides the package name.
+- `-d, --database <string>`: The name of the Postgres database instance to which this application will connect.
```

Comment:
For both these, clarify they are set in the initial deploy and do not change afterwards.